packages/vue/src/components/atoms/SfCircleIcon/SfCircleIcon.js

```diff
@@ -21,6 +21,13 @@ export default {
     sizeIcon: {
       type: String,
       default: "100%"
+    },
+    /**
+     * Sets aria-label for button icon
+     */
+    alt: {
```

Comment:
@mayashavin wdyt about this prop name for an `aria-label`? I'd be confused. imho `ariaLabel` would be clearer.

Reply:
@EduardDopler Indeed, miss that out.
@aniamusial please change it to `ariaLabel`. thanks

Reply:
Done :)